modules/custom/openy_addthis/openy_addthis.install

```diff
@@ -24,7 +24,7 @@ function openy_addthis_uninstall() {
     ->delete();
 
   // Remove our base field from the schema.
-  \Drupal::service('entity.definition_update_manager')->applyUpdates();
+  // TODO: \Drupal::service('entity.definition_update_manager')->applyUpdates();
```

Comment:
I think there should be code according to https://www.drupal.org/node/3034742

> Whenever an entity type or field storage definition needs to be created, changed or deleted, it has to be done with an explicit update function

Reply:
https://github.com/ymcatwincities/openy/pull/2174/commits/4e09e24c839250769eef5b4e73cf48b2c0331f01 fixed